src/browser/provider/built-in/base.js

```diff
@@ -15,10 +16,22 @@ function getCurrentSize () {
         height: window.innerHeight,
     };
 }
+
+function getMacSizeInfo () {
```

Comment:
BTW, can we get rid of duplication and just use `getSize` for this purpose (also rename it)?